src/components/blocks/ProductInfo/ProductInfo.stories.tsx

```diff
@@ -0,0 +1,48 @@
+// src/components/ProductInfo/ProductInfo.stories.tsx
+import { Meta, StoryFn } from '@storybook/react'
+import ProductInfo from './ProductInfo'
+import { descriptionBlob, infoBlob } from './testInfo'
+
+export default {
+	title: 'Components/ProductInfo',
+	component: ProductInfo,
```

Comment:
@elusiveuphoria for args that are complex like rich text, we don't need to have controls for those inside storybook.  Here is an example.  https://tinyurl.com/29w3bctc  We want to create argTypes, and do control false on the ones that we don't need storybook input field control over, most of the time callback functions etc fall under this scenario.   So in this case info, and description rich text blobs wouldn't be controlled by the storybook thingy.

Reply:
Done.